test/test_servers.py

```diff
@@ -0,0 +1,202 @@
+import pytest
+
+from click.testing import CliRunner
+
+from faculty_cli.cli import cli, _server_spec
+
+from faculty.clients.server import ServerClient
+from test.fixtures import (
+    PROJECT,
+    SHARED_SERVER,
+    DEDICATED_SERVER,
+    DEDICATED_RESOURCE,
+    SHARED_RESOURCE,
+    USER_ID,
+)
+
+
+@pytest.fixture
```

Comment:
Since these fixtures are identical to those in `test_projects`, can we factor them out into `tests.fixtures`?